libertree-frontend-ramaze/view/docs/en_GB/custom-css-file.markdown

```diff
@@ -0,0 +1,54 @@
+# Custom-CSS-File
+You can copy paste the code below in to your settings. Please put behind the semicolons a number in hexadecimal for the color. 
+
+##Clarification
+This file is dedicated to normal users, if you are familiar with CSS please take a look at the application.scss-file for advanced changes of the Libertree interface.
+
+## CSS-file ##
+
+~~~
+/* THIS IS THE EXAMPLE-CSS-FILE FOR LIBERTREE */
+
+/* Hint: If you want to change the colors, you have to edit the hexadecimal numbers*/
+
+/* In this section you can put a picture-link in for the background and you can choose a color if you picture doesn't cover the whole area*/
+
+body
+{
```

Comment:
Isn't this a matter of taste? IMHO as a beginner it's easier to see where a file starts and ends.